src/tests/ov_helpers/ov_snippets_models/src/subgraph_matmul.cpp

```diff
@@ -12,9 +12,76 @@
 namespace ov {
 namespace test {
 namespace snippets {
+namespace {
+constexpr float i8_fq_il = -35.0172004;
+constexpr float i8_fq_ih = 34.7436294;
+constexpr float u8_fq_il = 0;
+constexpr float u8_fq_ih = 0.820726;
```

Comment:
Probably we just need to create one more helper `create_fake_quantize` if we always create with the same weights?

Reply:
Good idea, thank you. Agree, let's create FQ builder

Reply:
The builder `make_fake_quantize` is created